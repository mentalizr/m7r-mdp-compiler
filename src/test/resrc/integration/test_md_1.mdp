### Header 1

Lorem ipsum dolor sit amet, consetetur saPlaudipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.

Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium:
- Ratgeber Depression: Informationen für Betroffene und Angehörige, M. Hautzinger, Hogrefe-Verlag, ISBN-10: 3801718794

#### Subheader 1
Here is no empty line after header.
Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt. Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit.

#### Subheader 2

At vero eos et accusamus et iusto odio dignissimos ducimus.

Klicken Sie auf *Weiter* um zum nächsten Schritt zu gelangen.

* Erster Punkt mit *kursivem* Inhalt
* Zweiter Punkt mit Inhalt in **bold**
* Dritter Punkt

Hier was in *kursiv*, **bold** und ***beides***. Hier `code`.

---

Hier noch was ...

| Header 1 | Header 2 | Header 3 |
| ---- | ---- | ---- |
| C1R1 | C2R1 | C3R1 |
| C1R2 | C2R2 | C3R2 |
