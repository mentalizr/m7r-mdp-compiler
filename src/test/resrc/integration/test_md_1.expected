<p class="h3 mt-4 mb-4">Header 1</p>
<p>Lorem ipsum dolor sit amet, consetetur saPlaudipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.</p>
<p>Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium:
- Ratgeber Depression: Informationen für Betroffene und Angehörige, M. Hautzinger, Hogrefe-Verlag, ISBN-10: 3801718794</p>
<p class="h4 mt-4 mb-4">Subheader 1</p>
<p>Here is no empty line after header.
Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt. Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit.</p>
<p class="h4 mt-4 mb-4">Subheader 2</p>
<p>At vero eos et accusamus et iusto odio dignissimos ducimus.</p>
<p>Klicken Sie auf <em>Weiter</em> um zum nächsten Schritt zu gelangen.</p>
<ul class="mb-4">
    <li>Erster Punkt mit <em>kursivem</em> Inhalt</li>
    <li>Zweiter Punkt mit Inhalt in <strong>bold</strong></li>
    <li>Dritter Punkt</li>
</ul>
<p>Hier was in <em>kursiv</em>, <strong>bold</strong> und <em><strong>beides</strong></em>. Hier <code>code</code>.</p>
<hr/>
<p>Hier noch was ...</p>
<table class="table">
    <thead>
        <tr>
            <th scope="col">Header 1</th>
            <th scope="col">Header 2</th>
            <th scope="col">Header 3</th>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>C1R1</td>
            <td>C2R1</td>
            <td>C3R1</td>
        </tr>
        <tr>
            <td>C1R2</td>
            <td>C2R2</td>
            <td>C3R2</td>
        </tr>
    </tbody>
</table>